scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Can we use the variable name IM_SIZE = (1920, 1080) to be consistent with other scenarios</li><li>Is TRACK_BUFFER an important parameter? If not, can we just set+use the 300 as default in the respective function.</li><li>Looks like FRAME_RATE isn't used and can be deleted. Also not sure BASELINE_MODEL and FT_MODEL are needed.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/553/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
BASELINE_MODEL and FT_MODEL are required for `load`

Reply:
Sounds all good. We might remove TRACK_BUFFER later then, if it is overly specific for our toy dataset.